bot/src/discord_commands/jibiki_all_search.js

```diff
@@ -0,0 +1,254 @@
+const {throwPublicErrorInfo} = require('./../common/util/errors.js');
+const {Navigation, NavigationChapter} = require('monochrome-bot');
+const axios = require('axios');
+const constants = require('./../common/constants.js');
+
+const jibikiApiUri = 'https://api.jibiki.app';
+const maxFieldsPerPage = 8;
+
+module.exports = {
+  commandAliases: ['jibiki', 'jb'],
+  cooldown: 3,
+  uniqueId: 'best_dictionary',
+  shortDescription: 'Search Jibiki.',
```

Comment:
Suggestion: you might want to make this a little more descriptive. For example for the Jisho command it's "Search Jisho for an English or Japanese word.". This will appear in the help command.